pkg/simple/client/auditing/options.go

```diff
@@ -32,6 +32,9 @@ type Options struct {
 	// The batch interval of auditing events.
 	EventBatchInterval time.Duration `json:"eventBatchInterval" yaml:"eventBatchInterval"`
 	Host               string        `json:"host" yaml:"host"`
+	BasicAuth          bool          `json:"basicAuth" yaml:"basicAuth"`
```

Comment:
Why do we need `BasicAuth` here?  If username and password provided, doesn't that mean want to connect with credentials?

Reply:
We have discussed this when designing the configuration items in cc. In the end we reached an agreement to add the basicAuth field; for the same reason, we also use the same field here. Otherwise, when the users set basicAuth item to false in cc, but does not delete the value of the username field, the configuration will not take effect.